### ISSUE TITLE
`now alias` regression

### ISSUE THREAD

**Issue by mathiasbynens** [vercel/vercel#278]
Copied from #272.

Previously, running just `now alias` would alias the most recent deployment using the `alias` field in `package.json`. Since v4 that doesn’t work anymore.

I just tested it with the `aliases` field instead (per the above) but it still errors:

```
$ now alias
> Error! Please specify a valid subcommand: ls | set | rm

  𝚫 now alias <ls | set | rm> <deployment> <alias>

  Options:

    -h, --help              Output usage information
    -c FILE, --config=FILE  Config file
    -d, --debug             Debug mode [off]
    -t TOKEN, --token=TOKEN Login token

  Examples:

  – Lists all your aliases:

      $ now alias ls

  – Adds a new alias to my-api.now.sh:

      $ now alias set api-ownv3nc9f8.now.sh my-api.now.sh

      The `.now.sh` suffix can be ommited:

      $ now alias set api-ownv3nc9f8 my-api

      The deployment id can be used as the source:

      $ now alias set deploymentId my-alias

      Custom domains work as alias targets:

      $ now alias set api-ownv3nc9f8.now.sh my-api.com

      – The subcommand `set` is the default and can be skipped.
      – `http(s)://` in the URLs is unneeded / ignored.

  – Removing an alias:

      $ now alias rm aliasId

      To get the list of alias ids, use `now alias ls`.

  Alias: ln
```

TL;DR `now alias` (without further parameters) regressed in v4.

**Comment by Rowno** [comment]
FYI, this was fixed in [4.3.0](https://github.com/zeit/now-cli/releases/tag/4.3.0) and the config field was changed from `aliases` to `alias`.
